iroh-relay/src/client.rs

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
(I'm lacking a good place to comment)

Same reasoning as in the other comment: The module comment is now much more prominent (just a click away from the top-level iroh-relay crate docs), I think even just something like "Code for establishing connections to a relay server" is more useful.